### PR TITLE
feature/1503 - f1m print preview

### PR DESCRIPTION
**Pull request by dheitzer**
I had to update the F1M dot_fec schema fields to get things working, but from referencing the spec I think this is right.

https://app.zenhub.com/workspaces/fecfile-online-619e578e68408b001c831251/issues/gh/fecgov/fecfile-web-app/1503

### CODE REVIEW

File: django-backend/fecfiler/web_services/dot_fec/schema_fields/F1M.json [hunk 1]
```diff
@@ -28,8 +31,8 @@
         "path": "form_1m.affiliated_committee_name"
     },
 
-    "I_candidate_id_number": {
-        "path": "form_1m.I_candidate_id_number"
+    "I_candidate_id": {
+        "path": "form_1m.I_candidate_id"
```
Comment (mjtravers) [inline django-backend/fecfiler/web_services/dot_fec/schema_fields/F1M.json:35]: @dheitzer Are yo sure about this change? In the model I see: I_candidate_id_number

Comment (dheitzer) [reply django-backend/fecfiler/web_services/dot_fec/schema_fields/F1M.json]: @mjtravers nice catch.  I pushed a fix but can only fully test when 1555 goes through.

Comment (toddlees) [reply django-backend/fecfiler/web_services/dot_fec/schema_fields/F1M.json]: add it to the list of field names we'd love to change if we could change the names in the spec

File: django-backend/fecfiler/web_services/dot_fec/schema_fields/F1M.json [hunk 2]
```diff
@@ -28,7 +31,7 @@
         "path": "form_1m.affiliated_committee_name"
     },
 
-    "I_candidate_id_number": {
+    "I_candidate_id": {
```
Comment (toddlees) [inline django-backend/fecfiler/web_services/dot_fec/schema_fields/F1M.json:34]: this field name needs to change to `I_candidate_id_number`

**Review by toddlees — CHANGES_REQUESTED** [review]

**Comment by mjtravers** [comment]
Changes for this ticket merged in with this PR: https://github.com/fecgov/fecfile-web-api/pull/667

**Pull request closed without merging**